pkg/util/util.go

```diff
@@ -30,9 +30,11 @@ var (
 	MaximumVolumeNameSize = 64
 	validVolumeName       = regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9_.-]+$`)
 
-	cmdTimeout = time.Minute // one minute by default
-
 	HostProc = "/host/proc"
+
+	DefaultVolumeSectorSize  = int64(4096)
+	DefaultReplicaSectorSize = int64(512)
```

Comment:
I am thinking what value we should use for replica sector size... Still 512?

Reply:
512 is fine in my test, but I cannot figure out why we need a 512 replica sector size before from the codes. Do you have any idea?

Reply:
No for me either. Actually, I prefer to use 4k here.